numba/cuda/tests/nocuda/test_function_resolution.py

```diff
@@ -0,0 +1,35 @@
+from numba.cuda.testing import unittest, skip_on_cudasim
+import operator
+from numba.core import types, typing
```

Comment:
For suggestion below:

```suggestion
from numba.core import types, typing
from numba.cuda.cudadrv import nvvm
```